Create README.md

inyt opened the pull request:
**#WHAT**
README.md作成、データベース設計を記載
README.rdoc削除

**#WHY**
データベース設計のため
マークダウン記法で見やすくするため

Commit:
create pull request

Commit:
Create README.md

hand12 commented on README.md:
```diff
@@ -0,0 +1,46 @@
+| colomn | type | option |
+|:-:|:-:|:-:|
+| name | string | unique: true, add_index |
+| description | text | |
```
今回はすでにあるアプリのデモをつくるので、お手本のアプリを忠実に再現しましょう！
見本のアプリにはこのような表記はないです。

inyt replied:
descriptionカラムを削除しました。

hand12 commented on README.md:
```diff
@@ -0,0 +1,46 @@
+- has_many :users, through: :group_users
+- has_many :messages
+
+### group_users
```
中間テーブルの名前には、ルールがあります。そのルールに従って名前をつけましょう！

inyt replied:
テーブル名をgroups_usersに変更しました。

hand12 commented on README.md:
```diff
@@ -0,0 +1,46 @@
+
+| colomn | type | option |
+|:-:|:-:|:-:|
+| body | text | null: false, add_index |
```
画像を保存する時、メッセージのbodyカラムにはあたいは何が入るかを考えてみてください！

inyt replied:
画像保存のみのメッセージが保存できるように、
bodyカラムのnull: falseを削除しました。

hand12 reviewed (commented):
おしいですね。いくつかおコメントしたのでご確認ください！

Commit:
Modify README.md

1. groupsテーブルからdescriptionカラム削除
2. テーブル名group_usersからgroups_usersに変更
3. messagesテーブルのbodyカラムのnull: false削除

inyt commented:
1. descriptionカラムを削除しました。
2. テーブル名をgroups_usersに変更しました
3. 画像保存のみのメッセージが保存できるように、
   bodyカラムのnull: falseを削除しました。
